roles/ovirt-ova-export-pre-pack/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+    - name: Examine target directory
```

Comment:
Any reason to use indent here? I think it's OK, to be at index 0.

Why note do all those checks in the script? If the script is already there, it will be more effecient.

Reply:
indentation - fixed

in the python script? because I call the playbook twice - once in the validation/can-do-action phase of the command in the action with validate_only='True' so only those checks will be executed (without the packing phase). if I move them to the script, I'll need to pass that parameter to the script. I believe that in some phase we will be required to report those errors to the client so it will introduce another complexity - passing them from the python script to ansible